lib/typescript/render/providers/chatplugin/ChatPluginRender.tsx

```diff
@@ -1,24 +1,61 @@
 import React from 'react';
```

Comment:
Do we really want to support rich cards in the chat plugin or was this just for developing it?

Reply:
The rich cards are actually not tied to Google (we are copying Google's structure and style)

Reply:
Ah, then all is good/